praw/models/user.py

```diff
@@ -130,9 +130,24 @@ def me(
             the cached value. Prefer using separate Reddit instances, however, for
             distinct authorizations.
 
+        .. deprecated:: 7.2
+
+            In read-only mode this method returns ``None``. In PRAW 8 this method will
+            raise :class:`.ReadOnlyException` when called in read-only mode. To operate
+            in PRAW 8 mode, set the config variable ``praw8_raise_exception_on_me`` to True.
```

Comment:
```suggestion
            In :attr:`.read_only` mode this method returns ``None``. In PRAW 8 this
            method will raise :class:`.ReadOnlyException` when called in
            :attr:`.read_only` mode. To operate in PRAW 8 mode, set the config variable
            ``praw8_raise_exception_on_me`` to ``True``.
```
Could we reference `read_only` and make `True` an inline literal?